src/main.js

```diff
@@ -108,7 +108,7 @@ if (categorySearch) categorySearchData.push(categorySearch);
 // setup fill color
 // colorColumn stores the name of column according to which coloring is to be done
 // check whether the coloring column is provided in the url or not
-let colorColumn = queryParams.get("color") || "Select";
+//let colorColumn = queryParams.get("color") || "Select";
```

Comment:
let's delete lines that are commented out if they're not used. Also we can kill the explanation comments if they're not relevant anymore.